### PR TITLE
PWN-6722 - Removed release local and firebase types. 

### PR DESCRIPTION
**Pull request by 1sakov**
Torus will be release store everytime

## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6722

## Description of Work

[ what work was done in this ticket ]

## Screenshots (Optional)

Screenshot #1 | Screenshot #2 | Screenshot #3
:------------:|:-------------:|:------------:
[Paste Pic 1] | [Paste Pic 2] | [Paste Pic 3]

### CODE REVIEW

File: .github/workflows/android-release-workflow.yml [hunk 1]
```diff
@@ -151,12 +44,17 @@ jobs:
...
           lokalise-app-id: ${{ secrets.LOKALISE_APP_ID }}
-      - name: "Build release AAB artifact"
+      - name: "Build release AAB&APK artifacts"
         env:
           KEYSTORE_PASSWORD: ${{ secrets.KEYSTORE_PASSWORD }}
           KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
           KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
-        run: ./gradlew bundleRelease
+        run: ./gradlew bundleRelease assembleRelease
+      - name: "Upload AAB to Firebase App Distribution"
```
Comment (gslevinkov) [inline .github/workflows/android-release-workflow.yml:53]: The release build that are published on Slack and Firebase won't work anymore, FYI

Comment (gslevinkov) [reply .github/workflows/android-release-workflow.yml:53]: how can we debug release builds on our local machines now?

Comment (1sakov) [reply .github/workflows/android-release-workflow.yml:53]: I know. QA's will sign via seed phrase

Comment (1sakov) [reply .github/workflows/android-release-workflow.yml:53]: You will debug by signing seed phrase as well

**Pull request merged**